docs/how-tos/hello-world-module.md

```diff
@@ -0,0 +1,508 @@
+
+1. Enter the path to the automatically-generated <file>run.sh</file> file, for example, `/Users/jessamyt/myCode/hello-world/run.sh`.
+   Click **Create**.
+
+1. Now add the modular camera resource provided by the module:
+
+   Click **+**, click **Local module**, then click **Local component**.
+
+   For the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet" >}}, enter `<namespace>:hello-world:hello-camera`, replacing `<namespace>` with the organization namespace you used when generating the stub files.
+   For example, `jessamy:hello-world:hello-camera`.
+
+   For type, enter `camera`.
+
+   For name, you can use the automatic `camera-1`.
+
+1. Configure the image path attribute by pasting the following in place of the `{}` brackets:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+    "image_path": "<replace with the path to your image>"
+   }
+   ```
+
+   Replace the path with the path to your image, for example `"/Users/jessamyt/Downloads/hello-world.jpg"`.
+
+1. Save the config, then click the **TEST** section of the camera's configuration card.
+
+   ![The Viam app configuration interface with the Test section of the camera card open, showing a hello world image.](/how-tos/hello-camera.png)
+
+   You should see your image displayed.
+   If not, check the **LOGS** tab for errors.
+
+{{< expand "Click if you also created a sensor component" >}}
+
+9. Add the modular sensor:
+
+   Click **+**, click **Local module**, then click **Local component**.
+
+   For the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet" >}}, enter `<namespace>:hello-world:hello-sensor`, replacing `<namespace>` with the organization namespace you used when generating the stub files.
+   For example, `jessamy:hello-world:hello-sensor`.
+
+   For type, enter `sensor`.
+
+   For name, you can use the automatic `sensor-1`.
+
+10. Save the config, then click **TEST** to see a random number generated every second.
+
+    ![The sensor card test section open.](/how-tos/hello-sensor.png)
+
+{{< /expand >}}
+
+## Package and upload the module
+
+You have a working local module.
+You can make it available to deploy on more machines by packaging it and uploading it to the [Viam registry](https://app.viam.com/registry).
+
+The hello world module you created is for learning purposes, not to provide any meaningful utility, so we recommend making it available only to machines within your {{< glossary_tooltip term_id="organization" text="organization" >}} instead of making it publicly available.
+
+{{< expand "Click to see what you would do differently if this wasn't just a hello world module" >}}
+
+1. Create a GitHub repo with all the source code for your module.
+   Add the link to that repo as the `url` in the <file>meta.json</file> file.
+1. Create a README to document what your module does and how to configure it.
```

Comment:
This is interesting. The generate command was supposed to also generate a readme. Please investigate whether this is an oversight

Reply:
Asking. The scope doesn't mention a readme though.